### ISSUE TITLE
A11y_mssql for VSCode_Select top 1000 _ScreenReader:When tabbing to the editor screen reader is not announcing the line number along with the data in the line.

### ISSUE THREAD

**Issue by Roopesh-T** [microsoft/vscode-mssql#17205]
## Actual:
When tabbing to the editor screen reader is not announcing the line number along with the data in the line.

**Similar issue is observed with Jaws**
#### Similar issue is also observed in following screens.

1. Script as create_editor
2. Script as Drop_editor
3. Script as Alter_editor

## Expected:
When tabbing to the editor screen reader should announce the line number along with the data in the line like line 1, line 2, line 3....

**GitHubTags:**#A11yMAS;#A11yTCS;#BM_mssql for VSCode_Jan2022;#mssql for VSCode;#Win32;#FTP;#DesktopApp;#A11ySev1;#AINotInScope;#WCAG1.3.1;#NVDA;#Jaws;#STP;#Element:editor;

## Environment Details:
Application Name: mssql for VSCode
Windows Version: Win11
Jaws version: 2021.2103.174 ILM
NVDA Version: Version 2021.3

## Repro Steps:
1. Open Visual studio code 
2. Tab till extension button and search for "mssql"
3. Tab till install and move to the SQL Server  button
4. Tab till Add Connection Button and insert the valid credentials
5. After adding the server move to database expand it -->System database expand-->Master expand-->Tables expand -->System table expand and right click on it and select top 1000 option
6. Select top 1000 window will open 
7. Run NVDA
8. Tab to the editor Check if the screen reader is announcing the position of the line along with the data in the line.

## User Impact:
Screen reader users will not get the position information in which screen reader focus is landing in editor.

“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”
## Attachment
![17205 1](https://user-images.githubusercontent.com/86601127/149275240-1ccfe457-a3d5-4710-8f8b-45f38af3b64f.gif)
![17205](https://user-images.githubusercontent.com/86601127/149275252-fce18da2-a364-4c48-8ab7-566d4ac17a05.gif)

**Comment by smartguest** [comment]
Closing as duplicate of #17205 (Same base vscode issue)

**Comment by smartguest** [comment]
Agreed, the line or row number needs to be announced somehow, this might require changes made to the query table to allow the screen reader to read the table's current row number, or perhaps the table class in general. Somewhere in the table there is some value or a series of values that the screen reader uses.

**Comment by smartguest** [comment]
The rowcount on a slickgrid table is labeled 0 which is incorrect, this needs to be updated to reflect the table more accurately

**Comment by smartguest** [comment]
Closing as this is mainly an upstream issue and is already closed for exclusion in ADS (for similar issues): 

https://github.com/microsoft/azuredatastudio/issues/6662
https://github.com/microsoft/azuredatastudio/issues/6688

**Comment by smartguest** [comment]
Need to investigate if this happens with vscode in general or just this extension.

**Comment by smartguest** [comment]
Closing as this is a general vscode issue with the editor and not one limited to the extension

**Comment by smartguest** [comment]
Relevant issues for the general bug in vscode:

https://github.com/microsoft/vscode/issues/52429

https://github.com/microsoft/monaco-editor/issues/410

**Comment by Roopesh-T** [comment]
As one of the external bugs is open https://github.com/microsoft/monaco-editor/issues/410 Please Open this bug and do not close this bug. Only a C & AI tester should close this bug.

**Comment by kburtram** [comment]
@alanrenmsft could you please follow-up what do with this issue?

**Comment by Roopesh-T** [comment]
@smartguest As one of the external bugs is open https://github.com/microsoft/monaco-editor/issues/410 Please Open this bug and do not close this bug. Only a C & AI tester should close this bug.

**Comment by alanrenmsft** [comment]
@Roopesh-T could you please follow up with vscode team?

**Comment by alanrenmsft** [comment]
@Roopesh-T this: https://github.com/microsoft/vscode/issues/52429 is the immediate upstream issue for this issue.

as mentioned in  https://github.com/microsoft/vscode/issues/52429#issuecomment-503161037, a workaround has been implemented, can we use it to mitigate the issue?
